src/benchmark-utils.ts

```diff
@@ -9,7 +9,23 @@ import {TensorFlowBenchmark} from './tfjs-benchmark';
 export interface BenchmarkResult {
   framework: string;
   backend: string;
-  duration: number;
+  actualBackend :string;
+  avg: number; 
+  pt50: number; 
+  pt95: number; 
+  pt99: number; 
+  min: number; 
+  max: number; 
+  webglPack : string;
```

Comment:
We don't use a lint for this repo, but it's better to keep styles as much as possible. I saw unnecessary whitespaces in this PR. Please, remove them and follow the existing coding styles.

If you can add a lint check, it would be the best. :)